Without arguments the command crashes

When executing `jx project` without arguments, it crashes:

```
➜ jx project
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x20 pc=0x21eb36e]

goroutine 1 [running]:
github.com/jenkins-x/jx-project/pkg/cmd/root.(*WizardOptions).Run(0xc0005980a0, 0xc0007b1cf8, 0x0)
/workspace/source/pkg/cmd/root/root.go:84 +0x3e
github.com/jenkins-x/jx-project/pkg/cmd/root.NewCmdMain.func1(0xc000202b00, 0x35f2768, 0x0, 0x0)
/workspace/source/pkg/cmd/root/root.go:69 +0x42
github.com/spf13/cobra.(*Command).execute(0xc000202b00, 0xc00009c190, 0x0, 0x0, 0xc000202b00, 0xc00009c190)
/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:846 +0x2aa
github.com/spf13/cobra.(*Command).ExecuteC(0xc000202b00, 0xc0005980a0, 0xc00006c750, 0xc0007b1f50)
/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:950 +0x349
github.com/spf13/cobra.(*Command).Execute(...)
/go/pkg/mod/github.com/spf13/cobra@v1.0.0/command.go:887
github.com/jenkins-x/jx-project/cmd/app.Run(0x0, 0x0, 0x0, 0x22d5ba0, 0xc00009a058)
/workspace/source/cmd/app/root.go:16 +0x80
main.main()
/workspace/source/cmd/main.go:11 +0x32
```

When executing it with `--help` for example, no problem:

```
➜ jx project --help
Create a new project by importing code, creating a quickstart or custom wizard for spring.

Usage:
 project [flags]
 project [command]

Examples:
 # Create a project using the wizard
 jx project

Available Commands:
 help        Help about any command
 import      Imports a local project or Git repository into Jenkins
 pullrequest Create a Pull Request on the git project for the current directory
 quickstart  Create a new app from a Quickstart and import the generated code into Git and Jenkins for CI/CD
 spring      Create a new Spring Boot application and import the generated code into Git and Jenkins for CI/CD

Flags:
 -h, --help   help for project

Use "project [command] --help" for more information about a command.
```

Before this, I've upgraded the CLI and the plugins:

```
➜ jx upgrade cli
about to run: git clone https://github.com/jenkins-x/jxr-versions.git /var/folders/m4/ytp3wjnn1hlbrxt27d64jc3c0000gn/T/jx-git-075610043 in dir /var/folders/m4/ytp3wjnn1hlbrxt27d64jc3c0000gn/T
Cloning into '/var/folders/m4/ytp3wjnn1hlbrxt27d64jc3c0000gn/T/jx-git-075610043'...
Jenkins X client has been upgraded to version 3.0.338

➜ jx upgrade plugins
checking binary jx plugin admin version 0.0.117 is installed
checking binary jx plugin application version 0.0.10 is installed
checking binary jx plugin gitops version 0.0.316 is installed
checking binary jx plugin jenkins version 0.0.22 is installed
checking binary jx plugin pipeline version 0.0.20 is installed
checking binary jx plugin preview version 0.0.82 is installed
checking binary jx plugin project version 0.0.99 is installed
checking binary jx plugin promote version 0.0.110 is installed
checking binary jx plugin secret version 0.0.144 is installed
about to run: /Users/hervelemeur/.jx3/plugins/bin/jx-secret-0.0.144 plugins upgrade
checking we have the correct vault CLI version
checking binary jx plugin test version 0.0.18 is installed
checking binary jx plugin verify version 0.0.26 is installed
```

Operating System   Mac OS X 10.15.6 build 19G2021

--- a/pkg/cmd/root/root.go
+++ b/pkg/cmd/root/root.go
@@ -6,6 +6,7 @@ import (
 
 	"github.com/jenkins-x/jx-helpers/pkg/cobras/helper"
 	"github.com/jenkins-x/jx-helpers/pkg/input"
+	"github.com/jenkins-x/jx-helpers/pkg/input/survey"
 	"github.com/jenkins-x/jx-project/pkg/cmd/common"
 	"github.com/jenkins-x/jx-project/pkg/cmd/importcmd"
 	"github.com/jenkins-x/jx-project/pkg/cmd/root/pullrequest"
@@ -81,6 +82,10 @@ func NewCmdMain() (*cobra.Command, *WizardOptions) {
 
 // Run implements the command
 func (o *WizardOptions) Run() error {
+	if o.Input == nil {
+		o.Input = survey.NewInput()
+	}
+
 	name, err := o.Input.PickNameWithDefault(createProjectNames, "Which kind of project you want to create: ",
 		"", "there are a number of different wizards for creating or importing new projects.")
 	if err != nil {

